dbt/models/default/schema/default.vw_pin_sale_combined.yml

```diff
@@ -0,0 +1,102 @@
+models:
+  - name: default.vw_pin_sale_combined
+    description: '{{ doc("view_vw_pin_sale_combined") }}'
+
+    columns:
+      - name: buyer_name
+        description: '{{ doc("shared_column_buyer_name") }}'
+      - name: class
+        description: '{{ doc("shared_column_class") }}'
+      - name: deed_type
+        description: '{{ doc("shared_column_deed_type") }}'
+      - name: doc_no
+        description: '{{ doc("shared_column_document_number") }}'
+      - name: is_multisale
+        description: '{{ doc("shared_column_sale_is_multisale") }}'
+      - name: is_mydec_date
+        description: Indicator for whether or not the observation uses the MyDec sale date
+      - name: mydec_deed_type
+        description: Deed type from MyDec, more granular than CCAO deed type
+      - name: nbhd
+        description: '{{ doc("shared_column_nbhd_code") }}'
+      - name: num_parcels_sale
+        description: '{{ doc("shared_column_num_parcels_sale") }}'
+      - name: pin
+        description: '{{ doc("shared_column_pin") }}'
+      - name: sale_date
+        description: '{{ doc("shared_column_sale_date") }}'
+      - name: sale_filter_deed_type
+        description: Remove quit claim, executor, beneficiary and missing deed types
+      - name: sale_filter_is_outlier
+        description: Same as `sv_is_outlier`, but `NULL` values are replaced with `FALSE`
+      - name: sale_filter_less_than_10k
+        description: Indicator for whether sale is less than $10K FMW
+      - name: sale_filter_same_sale_within_365
+        description: Remove sale with the _same_ value (for the same PIN) within 365 days
```

Comment:
I think you left out `sale_filter_same_iasworld_sale_within_365`. I would also mention each column in the description of each as they kind of exist in relation to each other.

`sale_filter_same_sale_within_365` uses sales from both wheras the other is only ias